packages/core/launch.ts

```diff
@@ -176,6 +176,11 @@ export interface LaunchOption {
      * @see {@link generateArguments}
      */
     prechecks?: LaunchPrecheck[];
+
+    /**
+    * The spawn process function. Used for spawn the java process at the end. By default, it will be the spawn function from "child_process" module. You can use this option to change the 3rd party spawn like [cross-spawn](https://www.npmjs.com/package/cross-spawn)
+    */
+   spawnFunction?: (command: string, args?: ReadonlyArray<string>, options?: SpawnOptions) => ChildProcess;
```

Comment:
can we just keep it `spawn`? (or I can merge the PR first and change it before publish)

Reply:
Yes, sure

Reply:
Also there are some interface also need to add this option. (So the compile will pass) 👍